main.go

```diff
@@ -299,12 +299,22 @@ func collectFilesToDeploy(absDeployPth string, config Config, tmpDir string) (fi
 
 func archiveAsTar(sourceDirPath, destinationTarPath string, isContentOnly bool) error {
 	fmt.Println()
-	log.Infof("Compressing directory...")
+	log.Infof("Archiving directory...")
+
+	// Note: The tar command can't change the pathname of the files
+	// when unarchiving so we need to avoid using the absolute path
+	// of the directory. Hence we set the working directory as the
+	// parent directory of the directory we're archiving.
+	// https://docstore.mik.ua/orelly/unix/upt/ch20_10.htm
+	workDir := strings.TrimRight(sourceDirPath, "/")
```

Comment:
Out of curiosity, how does this affect the `Dir` and `Base` functions below?  I'd expect that file path algorithms are savvy enough to ignore trailing `/` where appropriate.

Reply:
I tested in macOS and it made a difference whether or not there was a slash at the end.

Example:
Parent directory we want: `/var/folder/`
Target directory: `/var/folder/test`
Output of `Dir`: `/var/folder/`

Example2:
Parent directory we want: `/var/folder/`
Target directory: `/var/folder/test/`
Output of `Dir`: `/var/folder/test`

Reply:
Got rid of this in favor of using the `-C` argument with the `tar` command.